paperclip-storage-ftp.gemspec

```diff
@@ -20,4 +20,5 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency("rake")
   gem.add_development_dependency("daemon_controller", ">= 1.1.0")
   gem.add_development_dependency("activerecord")
+  gem.add_development_dependency("byebug") unless RUBY_PLATFORM =~ /java/
```

Comment:
Imho it doesn't hurt to add this dependency as the debugger is widely used and it's handy when the gem is already there and doesn't need to be installed and required manually. Nevertheless if that's really a pain point we can surely remove it.